sdk/communication/Azure.Communication.Administration/CHANGELOG.md

```diff
@@ -1,8 +1,31 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-11-16)
 ### Added
 - Support for mocking all client methods that use models with internal constructors
+- Added support for long-running operations. More detail under Breaking Changes.
+
+### Breaking
+- Support long running operations for purchasing phone numbers
```

Comment:
nit: end all sentences with a full stop.

Even the short "Renamed <> to <>." ones

Reply:
second nit: let's be consistent whether we write "long-running" or "long running", lines 6 and 9

Reply:
Done